src/components/landing/CenteredHero/HeroForm.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from 'react';
+import { Link, Button, TextField, Form } from 'components';
+
+import { FORM_NAMES, HONEYPOT_FIELD_NAME } from '../../../contactFormConstants';
+import { currentlyExecutingGitBranch } from '../../../environment';
+
+import { ScmToolSelect } from '../../forms/ScmToolRadioGroup';
+import { SubmissionSuccessModal } from 'components/FormSubmissionModal';
+
+const submitToNetlifyForms = async ({
+  email,
+  scmTool,
+  subToNewsletter,
+  netlifyFormName,
+  agreeToPolicies,
+  honeypotText,
+  submitButtonLabel = 'NOT_SUPPLIED',
+}) => {
+  const branch = currentlyExecutingGitBranch();
+
+  const formData = new FormData();
+  formData.append('form-name', netlifyFormName);
+  formData.append('email', email);
+  formData.append('scm-select', scmTool);
+  formData.append('sub-to-newsletter', subToNewsletter);
+  formData.append(HONEYPOT_FIELD_NAME, honeypotText);
+  formData.append('agree-to-policies', agreeToPolicies);
+  formData.append('deployed-branch', branch);
+  formData.append('submit-button-label', submitButtonLabel);
+
+  let resp;
+
+  try {
+    resp = await fetch('/', {
+      method: 'POST',
+      body: formData,
+    });
+  } catch (error) {
+    console.error('Submission failed', error, resp);
+  }
+
+  return resp;
+};
+
+const SubmissionSuccessPositiveBody = () => (
+  <p>
+    Thank you for requesting a free trial of Roadie Backstage. We&apos;ll be in touch via the email
+    provided.
+  </p>
+);
+
+const SubmissionSuccessNegativeBody = () => (
+  <>
+    <p>Roadie only supports GitHub for now.</p>
+    <p>
+      We are working to support more tools in the near future. You will be among the first to know
+      when we support yours.
```

Comment:
One thing that has happened before is that I received an angry email from a person who submitted this form after picking "Bitbucket", and were then told that we don't support bitbucket.

They felt that they had been tricked into supplying their email address, even though we didn't support their tool.

If we can prevent people feeling this way without making the form too ugly I think it would be beneficial.